assets/components/svg/svg.jsx

```diff
@@ -850,9 +923,18 @@ function SvgThankYouHeroMobile() {
       preserveAspectRatio="xMinYMid"
     >
       <g fill="#CE147C" fillRule="evenodd" transform="rotate(-148 126.06 183.951)">
-        <path d="M56.344 127.9c2.87-8.712 7.982-16.816 15.287-23.393 16.314-14.69 39.06-17.704 57.993-9.582 5.92-17.965 2.31-38.513-11.254-53.577-20.14-22.368-54.6-24.174-76.969-4.033-22.368 20.14-24.174 54.6-4.033 76.969a54.297 54.297 0 0 0 18.976 13.615z" opacity=".2" />
-        <path d="M108.34 199.367c.124-16.384 7.606-32.528 21.702-43.15a54.256 54.256 0 0 1 32.176-10.975 54.262 54.262 0 0 0-10.976-33.174c-18.115-24.038-52.286-28.84-76.325-10.726-24.038 18.114-28.841 52.286-10.727 76.324 10.838 14.383 27.424 21.88 44.15 21.701z" opacity=".4" />
-        <path d="M216.938 198.218c-16.643 10.6-25.652 28.838-25.193 47.357-.405.258-.815.511-1.229.76-25.8 15.503-59.283 7.154-74.785-18.646-15.502-25.8-7.154-59.283 18.646-74.785 25.8-15.502 59.283-7.154 74.785 18.646 5.031 8.373 7.55 17.555 7.776 26.668z" opacity=".6" />
+        <path
```

Comment:
are we changing the SVG here or is just formatting adjustment?

Reply:
Just formatting